all the sourcecode should be linted on CI build

v3.0.0 fails the build as follows:

```
Running "eslint:visualTests" (eslint) task

/Users/colineberhardt/Projects/d3-financial-components/visual-tests/indicator/forceIndex.js
  23:78  error  Missing semicolon  semi

✖ 1 problem (1 error, 0 warnings)
```

This is because the `ci` build only lints the components.

@bjedrzejewski which editor are you using? I'd recommend using one that has an eslint plugin so that you get errors reported whilst developing rather than relying on the build.

--- a/Gruntfile.js
+++ b/Gruntfile.js
@@ -303,7 +303,8 @@ module.exports = function(grunt) {
         ['connect:site', 'browserstacktunnel-wrapper', 'webdriver'] : []);
     grunt.registerTask('webdriverTests', ['eslint:webdriverTests', 'webdriverTests:browserstack']);
 
-    grunt.registerTask('ci', ['components', 'uglify:components', 'site', 'uglify:site', 'webdriverTests']);
+    grunt.registerTask('ci', ['eslint:visualTests', 'components', 'uglify:components', 'site',
+        'uglify:site', 'webdriverTests']);
 
     grunt.registerTask('default', ['watch:components']);
 };

--- a/visual-tests/indicator/forceIndex.js
+++ b/visual-tests/indicator/forceIndex.js
@@ -20,7 +20,7 @@
     forceAlgorithm(data);
 
     // Calculate the maximum absolute value of the index
-    var absoluteMax = d3.max(data, function(d) { return Math.abs(d.force); })
+    var absoluteMax = d3.max(data, function(d) { return Math.abs(d.force); });
 
     //Scailing the display using the maximum absolute value of the Index
     var yScale = d3.scale.linear()

